### PR TITLE
Rename cookbook to how to, update docs link

### PR DESCRIPTION
**Pull request by jtoar**
Corresponding changes to the site for the changes in the docs (still haven't pushed those changes up yet). Wait to merge!

### CODE REVIEW

File: code/html/_nav.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
           d="M12 6c1.11 0 2-.9 2-2 0-.38-.1-.73-.29-1.03L12 0l-1.71 2.97c-.19.3-.29.65-.29 1.03 0 1.1.9 2 2 2zm6 3h-5V7h-2v2H6c-1.66 0-3 1.34-3 3v9c0 .55.45 1 1 1h16c.55 0 1-.45 1-1v-9c0-1.66-1.34-3-3-3zm1 11H5v-3c.9-.01 1.76-.37 2.4-1.01l1.09-1.07 1.07 1.07c1.31 1.31 3.59 1.3 4.89 0l1.08-1.07 1.07 1.07c.64.64 1.5 1 2.4 1.01v3zm0-4.5c-.51-.01-.99-.2-1.35-.57l-2.13-2.13-2.14 2.13c-.74.74-2.03.74-2.77 0L8.48 12.8l-2.14 2.13c-.35.36-.83.56-1.34.57V12c0-.55.45-1 1-1h12c.55 0 1 .45 1 1v3.5z"
         />
       </svg>
-      <span class="ml-2">Cookbook</span>
+      <span class="ml-2" style="transform: translateY(2px);">How To</span>
```
Comment (jtoar) [inline code/html/_nav.html:65]: Optical centering

File: code/html/_nav.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
   </li>
   <li class="mb-4">
     <a
-      href="/docs/index"
+      href="/docs/intro"
```
Comment (jtoar) [inline code/html/_nav.html:19]:
```suggestion
      href="/docs/introduction"
```

**Comment by thedavidprice** [comment]
@jtoar are you adding redirects when you make changes like this? Please do if not as this breaks a couple years worth of back links in some cases... 😬

(I'm assuming we can just /cookbooks/* --> /how-to/*)

**Comment by jtoar** [comment]
@thedavidprice I did here, copying what we did for docs: https://github.com/redwoodjs/redwoodjs.com/commit/956472cc49822cf27bf0565cb84f5fc8f0ac2abb

**Comment by thedavidprice** [comment]
Perfect! Thank you for confirming

**Pull request merged**